baselines/webworker.generated.d.ts

```diff
@@ -2095,6 +2274,10 @@ type HeadersInit = Headers | string[][] | Record<string, string>;
 type BodyInit = Blob | BufferSource | FormData | URLSearchParams | ReadableStream | string;
 type RequestInfo = Request | string;
 type BlobPart = BufferSource | Blob | string;
+type HTMLOrSVGImageElement = never | never;
```

Comment:
these `never` do not look right.. 

so few things, first i think `object` is more natural type here than `never`. and second, we should firer duplicates in a union. so you do not get `object | object` or `object | BodyInit`